tests/defaults_test.go

```diff
@@ -25,5 +27,45 @@ func TestDefault(t *testing.T) {
 	require.NoError(t, err, "converted map to coder values")
 	require.NotNil(t, values, "values must be non-nil")
 	coderd := values.Coderd
-	require.Equal(t, 1, *coderd.Replicas, "expected 1 replica by default")
+	require.Equal(t, int32(1), *coderd.Replicas, "expected 1 replica by default")
+}
+
+func TestOverwriteReplica(t *testing.T) {
+	t.Parallel()
+
+	// Given
+	// The default helm chart
+	chart, err := loader.LoadDir("..")
+	require.NoError(t, err, "loaded chart successfully")
+	require.NotNil(t, chart, "chart must be non-nil")
+
+	// When
+	// We overwrite the replicas value and render the chart
+	var ValuesToOverwrite = &CoderValues{
+		Coderd: &CoderdValues{
+			Replicas: pointer.Int32(3),
+		},
+	}
+
+	objs, err := RenderChart(chart, ValuesToOverwrite, nil, nil)
+	// TODO@jsjoeio - getting an error here
+	// error deserializing "coder/templates/coderd.yaml": yaml: line 9: mapping values are not allowed in this context
+	require.NoError(t, err, "failed to render chart")
```

Comment:
@jawnsy I feel like I'm _so_ close. I'm doing something wrong though and getting this error. Any ideas?

![image](https://user-images.githubusercontent.com/3806031/140814835-39825583-2a1a-4e56-b36d-d4de7dc6a512.png)